core/domain/feedback_services.py

```diff
@@ -964,9 +919,9 @@ def _add_message_to_email_buffer(
             other_recipient_ids, feedback_message_reference, old_status,
             new_status, exploration_id, has_suggestion)
 
-    if message_length > 0:
-        # Send feedback message email only if message text is non empty.
-        # It can be empty in the case when only status is changed.
+    if message_length:
+        # Send feedback message email only if message text is non empty. It can
```

Comment:
It --> The message text (otherwise, unclear if referent is the feedback message email)

(and maybe put the second sentence in parens)